docs/release-notes/release-notes-0.8.3.md

```diff
@@ -0,0 +1,253 @@
+Additionally, warning messages are given to the user, in particular for the case where they choose a custom change address that is of a different scriptPubKey type than the wallet; this can lead to undesirable scenarios for coinjoins.
+
+`ad8cd74` Enable external/custom change addresses.
+
+`b4ca42d` taker: Fix change validation logic
+
+`b25d03c` Fixes bug introduced in ad8cd74
+
+### 2e50d03 Fix signmessage to be Electrum-compatible
+
+Verification of message signatures against segwit addresses is not currently functional/possible in Core, and additionally the signing function used for messages in jmbitcoin, derived from coincurve, was not compatible with Electrum either. This commit uses the functionality in python-bitcointx's signmessage module, and now the wallet method `signmessage` creates signatures on messages against segwit addresses that are verifiable in Electrum.
+Note that a feature like this is useful, given compatibility with Electrum (tested as of 4.0.9), but it is somewhat of a stop-gap. See [BIP 322](https://github.com/bitcoin/bips/blob/master/bip-0322.mediawiki) for how things may develop in future.
+
+Also note that we still do not have a verify method implemented here, only a sign method.
+
+For more details on the history of this feature, see [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/841#issuecomment-808329156).
+
+### BIP78 via daemon
+
+Although for most this is an under-the-hood change only, with no functionality altered, for a subclass of users it's important. After these changes, the Tor onion service used by the receiver, and the sender's request via the SOCKS5 proxy, happen only via the daemon package, which (for those users) can be run on a separate machine or otherwise isolated instance. This brings the BIP78 implementation in line with the standard Coinjoin implementation with respect to network connections being isolate-able.
+
+`15468cb` Payjoin receiver via daemon.
+
+`b6e2576` BIP78 sender protocol via daemon
+
+`09b3fab` Remove Tor config defaults request in onion setup
+
+`8b0d08d` Don't request privkey for ephemeral onion from Tor
+
+### Change commitments logic to avoid using different mixdepths' utxos
+
+See [PR 840](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/840). This represents a fix to a minor hole in the privacy of takers. Before this change, takers would occasionally use utxos for PoDLE commitments that were not in the same mixdepth as their inputs for the coinjoin (first, utxos in the transaction were preferred for PoDLE creation; then, . Where this happened, the makers who received the opening of the PoDLE commitment, which includes the utxo, may be able to correlate these utxos with those in the final coinjoin transaction which they deduced to belong to the taker, and thus make a high-likelihood correlation of utxos between different mixdepths, something that Joinmarket tries hard to avoid.
```

Comment:
```suggestion
See [PR 840](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/840). This represents a fix to a minor hole in the privacy of takers. Before this change, takers would occasionally use utxos for PoDLE commitments that were not in the same mixdepth as their inputs for the coinjoin (first, utxos in the transaction were preferred for PoDLE creation; then, if those were not eligible, any other utxo in the wallet was tried). Where this happened, the makers who received the opening of the PoDLE commitment, which includes the utxo, may be able to correlate these utxos with those in the final coinjoin transaction which they deduced to belong to the taker, and thus make a high-likelihood correlation of utxos between different mixdepths, something that Joinmarket tries hard to avoid.
```

The closing parenthesis is missing and the sentence seems hanging, I added a possible interpretation but IDK what was the idea here.

Reply:
Yes, thanks. It looks like there was an accidental delete; indeed that's what I remember writing.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/912/commits/f1e440cd596e5d46d9da9d6bea0db9e1489d988d